src/helper-classes/position.scss

```diff
@@ -0,0 +1,27 @@
+/**
+* Child Alignment
+* `alignChild--<horizontal alignment>--<vertical alignment>`
+*/
+$xAlign: (
+  left: flex-start,
+  center: center,
+  right: flex-end,
+);
+$yAlign: (
+  top: flex-start,
+  center: center,
+  bottom: flex-end,
+);
+
+[class^="alignChild--"] {
+  display: flex;
+}
```

Comment:
wow, so this styles the class above? I didn't know that was possible even with sass